packages/yum/install_test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exu
+
+package="$1"
+
+os_version=$(cat /etc/almalinux-release | cut -f3 -d' ' | cut -f1 -d'.')
```

Comment:
最初の `cat` が不要かもしれないです。
```suggestion
os_version=$(cut -f3 -d' ' /etc/almalinux-release | cut -f1 -d'.')
```

OSバージョンの取得には `/etc/system-release-cpe` を使うことが多い印象です。
`cut -d: -f5 /etc/system-release-cpe` 文字数も減ります。

Reply:
たしかに。ありがとうございます。`/etc/system-release-cpe`を使うようにします。

Reply:
変更後のテストが通ることを確認しました。
https://github.com/komainu8/mroonga/actions/runs/12667581419/job/35301228150